docs/content/concepts/accounts-and-keys.md

```diff
@@ -82,6 +82,8 @@ Flow will have initial support for a predefined set of signature and hash pairin
 | ECDSA     | P-256     | ECDSA_P256      | 2    |
 | ECDSA     | secp256k1 | ECDSA_secp256k1 | 3    |
 
+*Please node that codes used for the keys here are different from the ones [defined in Cadence](https://docs.onflow.org/cadence/language/crypto/#signing-algorithms)*
```

Comment:
The Access API documentation refers to this page (https://docs.onflow.org/access-api/#account-key)